packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -1,12 +1,33 @@
-import { ReactNode } from 'react';
+import {
+  type ReactNode,
+  type MouseEvent,
+  type KeyboardEvent,
+  useRef,
+  useCallback,
+} from 'react';
 import { css } from '@emotion/react';
-import Constraints from '@commercetools-uikit/constraints';
 import { designTokens } from '@commercetools-uikit/design-system';
 import * as Popover from '@radix-ui/react-popover';
 import { Footer } from './footer';
 import { Header } from './header';
 import { TriggerButton } from './trigger-button';
 
+/**
+ * CSS selector to find focusable elements.
+ * @see https://github.com/microsoft/tabster/blob/6bfd54a45f5b20eccd17b8a05f6c86c241b992c3/src/Focusable.ts#L17-L25
+ */
+const FOCUSABLE_CSS_SELECTOR = `a[href], button:not([disabled]), input:not([disabled]), select:not([disabled]), textarea:not([disabled]), *[tabindex], *[contenteditable]`;
```

Comment:
Looks terrible 😄 but certainly an interesting solution. Can you add a TODO comment, so that we can find it easier and abstract away if it works reliably (this could certainly be reused for all kind of modals)